custom_components/plugwise/const.py

```diff
@@ -367,137 +368,133 @@
 
 # --- Const for Plugwise USB-stick.
 
-PLATFORMS_USB = ["binary_sensor", "sensor", "switch"]
+PLATFORMS_USB = [BINARY_SENSOR_DOMAIN, SENSOR_DOMAIN, SWITCH_DOMAIN]
 CONF_USB_PATH = "usb_path"
 
 # Callback types
 CB_NEW_NODE = "NEW_NODE"
+CB_JOIN_REQUEST = "JOIN_REQUEST"
 
 # Sensor IDs
-AVAILABLE_SENSOR_ID = "available"
-CURRENT_POWER_SENSOR_ID = "power_1s"
-TODAY_ENERGY_SENSOR_ID = "power_con_today"
-MOTION_SENSOR_ID = "motion"
+USB_AVAILABLE_ID = "available"
+USB_CURRENT_POWER_ID = "power_1s"
+USB_CURRENT_POWER_8S_ID = "power_8s"
+USB_POWER_CONSUMPTION_TODAY_ID = "power_con_today"
+USB_MOTION_ID = "motion"
+USB_RELAY_ID = "relay"
 
 # Sensor types
-USB_SENSORS = {
-    AVAILABLE_SENSOR_ID: {
+STICK_API = {
+    USB_AVAILABLE_ID: {
```

Comment:
Why this construct? Seems unnecessary complex? But maybe I'm not seeing the big picture...

Reply:
I removed the `USB_AVAILABLE_ID` construct in [62a5a7e](https://github.com/plugwise/plugwise-beta/pull/147/commits/62a5a7e1fbe0362c68033f817034722de1962094)